CMSgov/design-system
# [WNMGDS-2843] Add `ds-text-field` component

## Summary

[WNMGDS-2843](https://jira.cms.gov/browse/WNMGDS-2843)

Adds a new `ds-text-field` web component.

## How to test

1. Try it out in storybook
2. If you really want to, you can try adding it to the astro example too, but I did that myself already and removed it because I didn't have a good use case for it in the example, but maybe that's lazy and I should actually just revamp the example page and make a more meaningful form example that includes more components.

## Checklist


### If this is a change to code:


### If this is a change to documentation:
